data/registry/collector-receiver-database.yml

```diff
@@ -5,10 +5,11 @@ language: java
 tags:
   - receiver
   - database
-  - semantic convention 
+  - semantic convention
```

Comment:
Why "semantic convention"?

Reply:
We have proposed semantic convention for database under discussions with community.
https://github.com/instana/otel-database-dc/blob/main/docs/semconv/database.md

Reply:
The tag is not very required because it is not the major purpose of this tool. I just removed the tag "semantic convention". I have add another tag "instana" which is the IBM product supporting this tool.